components/02-molecules/menu/_menu-item.twig

```diff
@@ -14,29 +19,44 @@
   {% set item__modifiers = item__modifiers|merge(['with-sub']) %}
 {% endif %}
 
-{% import "@molecules/menu/menu.twig" as menus %}
+{# @TODO: When we build this in Drupal. Verify this works #}
+{# @LINK: https://yaleits.atlassian.net/browse/YALB-352 #}
+{% if item.is_active %}
```

Comment:
I'm fairly certain the variable we're looking for here is `in_active_trail`

Reply:
From what I could find `in_active_trail` would apply to the current page _and_ all of it's parents, grandparents, etc. So it's not specifically the "active page". `is_active` is specifically the current page.